### ISSUE TITLE
*** No instances to display ***  on Google Colab

### ISSUE THREAD

**Issue by ddm06** [matterport/Mask_RCNN#2558]
I'm training a mask r cnn model to detect 13 class. When I'm testing the model, I get the following error: *** No instances to display *** .

I tried this changes:

```
> utils.py line 866: shift = np.array ([0, 0, 1, 1]) to shift = np.array ([0, 0, 1., 1.]);

> downgrade scipy to 1.2.3 version;

> set model.load_weights(weights_path, by_name=True, exclude=["mrcnn_class_logits", "mrcnn_bbox_fc", "mrcnn_bbox", "mrcnn_mask"]) ;

>  from __future__ import division
```

Sometimes them work, other times they don't.

I have another question: I have 21 classes but i want to clasify only 13 of them. My goals are:

>  detect this 13 classes inside a specif area (like a truck), how can I do this?

> i would like to predict all items in a single image in maximum 1 second, how can I do this?

**Comment by TimNagle-McNaughton** [comment]
Should really post a thread for each question here. If there are no instances the model is not trained, could be a problem with managing your data or a lack of training.
